horace_core/sqw/@rundatah/private/calc_projections_.m

```diff
@@ -26,7 +26,7 @@
 %              This matrix can be used to convert components of a vector in
 %              crystal Cartesian axes to r.l.u.: v_rlu = u_to_rlu * v_crystal_Cart
 %              (Same as inv(B) in Busing and Levy convention)
-%   urange      [2 x 4] array containing the full extent of the data in crystal Cartesian
+%   pix_range  [2 x 4] array containing the full extent of the data in crystal Cartesian
```

Comment:
the return value is img_range not pix_range

Reply:
see above